src/Elasticsearch/Product/ElasticsearchProductDefinition.php

```diff
@@ -395,6 +399,8 @@ private function fetchProducts(array $ids, Context $context): array
 SELECT
     LOWER(HEX(p.id)) AS id,
     IFNULL(p.active, pp.active) AS active,
+    p.available AS available,
+    IFNULL(p.is_closeout, pp.is_closeout) AS isCloseout,
```

Comment:
This is now duplicate

Reply:
What do you mean?

Reply:
isCloseout is selected twice in the query now

Reply:
Oh - I checked for available only 😂 Thanks for pointing this out!